docs/quality-assurance/safety-assurance.md

```diff
@@ -0,0 +1,98 @@
+# What is safety ?
+
+So the first question is what is safety in relation to UKHO products:
```

Comment:
Shall we not start sentences with "so" :)

Reply:
Definitely agree!